newsrevenuehub/rev-engine
# [NRH-138] PayFees changes

#### What's this PR do?
- Move's the "Pay transaction fees" button down into the payment widget.
- Fixes the calculated fee to include fee increase from original payment + fee

#### How should this be manually tested?
- Visit a live donation page
- Make sure that donation page is set up to display the "Pay transaction fees" button
- Verify that the pay transaction fees toggle is grouped with the payment widget
- Select an amount to donate and verify that the correct fee is being applied
- Complete the contribution and verify that the correct amount is recorded on the Contribution object in Django-admin

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No
